src/get_env.c

```diff
@@ -22,14 +22,7 @@ extern "C"
 
 #include "rcutils/get_env.h"
 
-#ifdef _WIN32
-# include <errno.h>
-// all environment variables live together in a single memory block
-// which has a limit of 32767 characters
-# define WINDOWS_ENV_BUFFER_SIZE 32767
-static char __env_buffer[WINDOWS_ENV_BUFFER_SIZE];
-#endif  // _WIN32
-
+#pragma warning(disable : 4996)
```

Comment:
Ah, CI is showing unknown pragmas from GCC and clang.  You'll need to wrap this in an `#ifdef _WIN32`.

Reply:
I've updated this bit and the comment